src/code_3bf0_len_6f0.c

```diff
@@ -1,6 +1,24 @@
 #include "common.h"
 
-INCLUDE_ASM("code_3bf0_len_6f0", func_800287F0);
+void func_800287F0(void) {
+    GameStatus* gameStatus = *gGameStatusPtr;
+    GameStatus* gameStatus2;
+    GameStatus* gameStatus3;
+
+    gameStatus->stickX = 0;
```

Comment:
does this require this order? If not, these really look like macros, and I'd suggest a comment saying as such for each

Reply:
There're definitely macros here, but I don't know what each one entails based on this function. And the order of stuff here was necesssary to match

Reply:
maybe make the macro use more clear with

```c
// TODO: refactor blocks into macros

{
    GameStatus* game_status = &gGameStatusPtr;
    // ...
}

{
    GameStatus* game_status = &gGameStatusPtr;
    // ...
}

{
    GameStatus* game_status = &gGameStatusPtr;
    // ...
}
```

Reply:
I can leave a comment, but I'm not sure how to block-ify what I have. The ordering is weird and seems necessary.